travellan/travellan-rn
# Feature/253 edit notes

## Changes made

- I've added icon for editing note
- I've created screen view for editing note (title and discription for editing notes from categories: 'Without Category", "Diaries", "To Do" and just description for "To Pack" category)
- I've created editing notes logic in actions and reducer
- I've fixed bug where you couldn't delete notes from three categories (expect To-Pack)
- I've fixed bug with wrong import of navigationButton (this bug appeard in develop branch)
- I've  created correct validation of eddited note
- I've used Prettier and ESLint


**THERE IS BUG that i couldn't fixed - In order to see edited note you have to open 'Notes' again. Propably some problem with states (old issue)**

## Screenshots

(prefer animated gif)

## Test path (if necessary)

1. Go to Notes screen
2. Click on edit button
3. Edit your note
4. Go out of notes screen
5. Open notes screen again in order to see changes
6. Try this for notes from every category
...

## Checklist

- [ ] merged develop branch into feature branch
  - [ ] in emulator
  - [x] on a real device
- [ ] added new dependencies (what deps?)
- [x] used ESLint (linted) and Prettier (formatted)
